src/utils/IO/system_checks.jl

```diff
@@ -61,3 +61,53 @@ function critical_components_check(sys::System)
     missing_critical_components &&
     throw(IS.InvalidValue("Critical Componeents are not present."))
 end
+
+"""
```

Comment:
If you want these docstrings to be proper you have to add the function signature to the top.

```
    adequacy_check(sys::System)

Checks the system for sum(generator ratings) >= sum(load ratings).

...
```